io/src/lib.rs

```diff
@@ -67,7 +67,7 @@ pub trait Read {
     ///
```

Comment:
The commit message suggests this enables `alloc` from `std` but it doesn't because it already was. This merely removes redundant conditions. (Not worth blocking, wrote for other reviewers.)

Reply:
In future if I do rebase fails (assuming that's what this was) and commit descriptions are totally wrong I'd prefer not to merge them. In the hope that we can maintain a half bisectable git index. Thanks for noticing.

Reply:
In this case I thought it's just inaccuracy that can be resolved simply by looking into `Cargo.toml`.